app/resources/views/pages/profile/profile.blade.php

```diff
@@ -121,7 +121,8 @@
 							<label for="province">State/Province</label>
 						</div>
 						<div class="col-sm-8 col-xs-7">
-							<input list="regions" class="form-control" id="home_region" name="home_region" value="{{ $user->home_region }}">
+							<select list="regions" class="form-control" id="home_region" name="home_region" data-value="{{ $user->home_region }}">
+							</select>
 							<datalist id="regions"></datalist>
```

Comment:
Any reason you're using a datalist instead of putting the options directly in the select element?  For an input, the datalist made sense but select can have multiple options directly inside it.